src/main/java/com/ecore/roles/service/UsersService.java

```diff
@@ -0,0 +1,13 @@
+package com.ecore.roles.service;
+
+import com.ecore.roles.client.model.User;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface UsersService {
+
+    User getUser(UUID id);
+
+    List<User> getUsers();
```

Comment:
It could be just called "get" or "getAll", because it's inside of the users service.